cluster/examples/kubernetes/ceph/common.yaml

```diff
@@ -120,6 +120,18 @@ spec:
     singular: cephfilesystem
   scope: Namespaced
   version: v1
+  validation:
```

Comment:
#3534 will create this validation, except without the minimum set. How about we merge that first, then rebase this?

Reply:
Sure. no problem. :-)